src/backend/joanie/core/models/products.py

```diff
@@ -248,6 +254,23 @@ def state(self) -> str:
         dates = self.get_equivalent_course_run_dates()
         return CourseRun.compute_state(**dates)
 
+    @property
+    def has_withdrawal_period(self):
```

Comment:
I feel like `allow_withdrawal` or `is_withdrawable` may be more clear, WDYT ?

Reply:
`is_withdrawable` looks great! 👍